### PR TITLE
Retire helm documentation as we use a protobuf

### PR DESCRIPTION
**Pull request by sdake**
The new rendered source of truth is:
https://preliminary.istio.io/docs/reference/config/istio.operator.v1alpha1/

This is rendered from the API repo protobuf which (may) need description fields
set. That protobuf is here:

https://github.com/istio/api/blob/master/operator/v1alpha1/operator.proto

### CODE REVIEW

**Comment by sdake** [comment]
every update of the common-files removes any local changes to this repository breaking local listing: https://github.com/istio/istio/issues/23065

**Comment by sdake** [comment]
Flowchart to remove content: https://preliminary.istio.io/about/contribute/remove-content/

File: content/en/docs/reference/config/policy-and-telemetry/mixer-overview/index.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ At a high level, Mixer provides:
 
 Policy enforcement and telemetry collection are entirely driven from configuration.
```
Comment (sdake) [inline content/en/docs/reference/config/policy-and-telemetry/mixer-overview/index.md:44]: @frankbu is grabbing the reference documentation part of the release process? I can't find any of this content in the API repository.

Cheers
-steve

File: content/en/blog/2019/performance-best-practices/index.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ Say you run 2,000 Envoy-injected pods, each handling 1,000 requests per second.
...
-1.  **Custom policy checks:** When using [custom Istio policy adapters](/docs/concepts/observability/), policy checks are on the request path. This means that request headers and metadata on the data path will be sent to the control plane (Mixer), resulting in higher request latency. **Note:** These policy checks are [disabled by default](/docs/reference/config/installation-options/#global-options), as the most common policy use case ([RBAC](https://archive.istio.io/1.4/docs/reference/config/security/istio.rbac.v1alpha1)) is performed entirely by the Envoy proxies.
+1.  **Custom policy checks:** When using [custom Istio policy adapters](/docs/concepts/observability/), policy checks are on the request path. This means that request headers and metadata on the data path will be sent to the control plane (Mixer), resulting in higher request latency. **Note:** These policy checks are [disabled by default](https://archive.istio.io/v1.4/docs/reference/config/installation-options/), as the most common policy use case ([RBAC](https://archive.istio.io/1.4/docs/reference/config/security/istio.rbac.v1alpha1)) is performed entirely by the Envoy proxies.
```
Comment (frankbu) [inline content/en/blog/2019/performance-best-practices/index.md:36]:
```suggestion
1.  **Custom policy checks:** When using [custom Istio policy adapters](/docs/concepts/observability/), policy checks are on the request path. This means that request headers and metadata on the data path will be sent to the control plane (Mixer), resulting in higher request latency. **Note:** These policy checks are [disabled by default](https://archive.istio.io/v1.4/docs/reference/config/installation-options/), as the most common policy use case ([RBAC](https://archive.istio.io/v1.4/docs/reference/config/security/istio.rbac.v1alpha1)) is performed entirely by the Envoy proxies.
```

**Pull request merged**